### ISSUE TITLE
Remote Archive SCM tooltip cleanup

### ISSUE THREAD

**Issue by dsesami** [ansible/awx#8140]
Link https://github.com/ansible/awx/issues/7954

##### ISSUE TYPE
 - Bug Report

##### SUMMARY
This tooltip (when selecting a remote archive type for a project) should have a black background like the rest of the UI. The links don't need to be clickable, as originally written, but perhaps they could be formatted somehow.
![Screenshot from 2020-09-14 15-46-02](https://user-images.githubusercontent.com/19942822/93133247-e9743d00-f6a4-11ea-936d-c0412cfdfc5b.png)

##### STEPS TO REPRODUCE
1. Go to Projects > Add
2. Set the type to "Remote Archive"

**Comment by unlikelyzero** [comment]
Covered in pull/5626
